ExposedAccessKeys/lambda_functions/notify_security.py

```diff
@@ -83,3 +100,32 @@ def publish_msg(subject, message):
         print(e)
         print('Could not publish message to SNS topic "{}"'.format(TOPIC_ARN))
         raise e
+
+print("============= Post to Slack ===========")
+
+def notify_slack(subject, subject2):
+
+   data = "{content:" + '"' + subject + subject2 +'"}'
+
+   headers = {
+      'Content-type': 'application/json'
+   }
+
+   ## USING Python "urllib" instead
+
+   data = data.encode('ascii')
+
+   headers = {}
+   headers['Content-Type'] = "application/json"
+
+   ## Send the request
+   print("URL = ", slack_webhook_url)
+   req = urllib.request.Request(slack_webhook_url, data=data, headers=headers)
+   resp = urllib.request.urlopen(req)
+
+   ## Receive the response
+   print("RESPONSE: ", resp)
+
+   return {
+     'statusCode': 200
+   }
```

Comment:
Return is not respected. 

Could we move this to the `lambda_handler` function? Otherwise you would have to change line 66 to:

```python
return notify_slack(subject, subject2)
```